src/PVRIptvData.h

```diff
@@ -111,11 +111,11 @@ class PVRIptvData : public P8PLATFORM::CThread
   virtual PVRIptvEpgChannel*   FindEpgForChannel(PVRIptvChannel &channel);
   virtual bool                 FindEpgGenre(const std::string& strGenre, int& iType, int& iSubType);
   virtual bool                 GzipInflate( const std::string &compressedBytes, std::string &uncompressedBytes);
-  virtual int                  GetCachedFileContents(const std::string &strCachedName, const std::string &strFilePath, 
+  virtual int                  GetCachedFileContents(const std::string &strCachedName, const std::string &strFilePath,
                                                      std::string &strContent, const bool bUseCache = false);
   virtual void                 ApplyChannelsLogos();
   virtual void                 ApplyChannelsLogosFromEPG();
-  virtual std::string          ReadMarkerValue(std::string &strLine, const char * strMarkerName);
+  virtual std::string          ReadMarkerValue(const std::string &strLine, const char * strMarkerName);
```

Comment:
no blank before &, but after

BTW: All this methods being `virtual` makes zero sense. But this is not for this PR.

Reply:
Will give it a refactor at some point

Reply:
👍 that would be highly appreciated